Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -76,6 +77,26 @@ public class TeleportPointer : LinePointer, IMixedRealityTeleportPointer, IMixed
         /// </summary>
         public DistorterGravity GravityDistorter => gravityDistorter;
 
+        private float cachedInputThreshold = 0f;
+
+        private float inputThresholdSquared = 0f;
+
+        /// <summary>
+        /// The square of the InputThreshold value.
+        /// </summary>
+        private float InputThresholdSquared
+        {
+            get
+            {
+                if (cachedInputThreshold != inputThreshold)
```

Comment:
May want to [use `Mathf.Approximately` here](https://docs.unity3d.com/ScriptReference/Mathf.Approximately.html).